libs/ui/inputs/src/lib/autocomplete/autocomplete.component.ts

```diff
@@ -109,6 +109,7 @@ export class AutocompleteComponent
   }
 
   ngAfterViewInit(): void {
+    this.inputRef.nativeElement.focus()
```

Comment:
I think you removed this line in the other commit, but you still need it otherwise the field is not focused